core/src/main/java/io/bisq/core/btc/wallet/WalletsSetup.java

```diff
@@ -86,7 +86,7 @@
     private final BisqEnvironment bisqEnvironment;
     private final BitcoinNodes bitcoinNodes;
     private final int numConnectionForBtc;
-    private final boolean useAllProvidedNodes;
+    private boolean useAllProvidedNodes;
```

Comment:
Sort mutable (non-final) fields after final fields so that the reader can more easily identify what's stateful and thus likely to be problematic.